Dockerfile

```
@@ -1,4 +1,4 @@
-FROM golang:1.16.6 AS builder
+FROM golang:1.17.3 AS builder
```

There is a newer version by now. Any reason for not using the latest release?

```suggestion
FROM golang:1.17.4 AS builder
```

---

Yes, I missed that .4 was released :D Thanks, adjusted it!